### ISSUE TITLE
Invalid window id when trying to edit files in floating window with codewindow.nvim open

### ISSUE THREAD

**Issue by nyngwang** [gorbit99/codewindow.nvim#8]
I got the following error message when I try to edit a buffer:

```
Error executing vim.schedule lua callback: ...k/packer/start/codewindow.nvim/lua/codewindow/window.lua:67: Invalid window id: 1026
stack traceback:
        [C]: in function 'nvim_win_get_buf'
        ...k/packer/start/codewindow.nvim/lua/codewindow/window.lua:67: in function ''
        vim/_editor.lua: in function ''
        vim/_editor.lua: in function <vim/_editor.lua:0>
```

Screencast:

<img src="https://user-images.githubusercontent.com/24765272/196338223-a9e40505-c284-4eb5-8458-867c520de10d.png" width="500">

**Comment by nyngwang** [comment]
I just worked around this by calling `close_minimap()` before creating the floating window and `open_minimap()` again when the floating window did get created. There is another related issue that I need to add

```lua
vim.api.nvim_feedkeys('jk', 'n', true)
```

so that the minimap will move to the right position of the floating window.

**Comment by dsully** [comment]
I'm seeing this too when using https://github.com/linty-org/key-menu.nvim and it opens a floating window.

**Comment by gorbit99** [comment]
Please check if this works now after the fix for #9

**Comment by dsully** [comment]
Working for me now. Thanks for the fix!
